### PR TITLE
Add documentation to support SigV4 serverless service as a data source connection for Dashboards

### LINKED ISSUE #3983: [DOC] [Multiple Datasource] Update documentation for AWS OpenSearch Serverlesss support in 2.8
**What do you want to do?**
 In upcoming 2.8 release, we plan to add support for AWS OpenSearch Serverlesss support, in multiple datasource. We need some updates to the documentation 

- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
https://opensearch.org/docs/latest/dashboards/discover/multi-data-sources/#creating-a-data-source-connection
image

Within the existing "aws sigV4" auth type configuration while creating a datasource, we will add an "service" option that user will  choose between "Amazon OpenSearch Service" or "Amazon OpenSearch Serverless"

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
- original feature request issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3623
- draft PR with video attached https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3957

### PR DESCRIPTION
**Pull request by cwillum**
### Description
A recent update includes support for the Amazon OpenSearch Serverless service for the SigV4 auth type. This supports the legacy js client.

### Issues Resolved
Updates documentation to cover this service as an option when users select SigV4 for auth type.

Fixes #3983 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by zhongnansu — APPROVED** [review]
lgtm!

**Comment by cwillum** [comment]
Will replace image once UI text is fixed.

File: _dashboards/discover/multi-data-sources.md [hunk 1]
```diff
@@ -39,16 +39,21 @@ To create a new data source connection:
...
     - **No authentication**: No authentication is used to connect to the data source.
     - **Username & Password**: A basic username and password are used to connect to the data source.
-    - **AWS Sigv4**: An AWS Signature Version 4 authenticating request is used to connect to the data source. AWS Sigv4 requires an access key and a secret key. First specify the **Region**, and then enter the **Access Key** and **Secret Key** for authorization. For information about available AWS Regions for AWS accounts, see [Available Regions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions). For more about Sigv4 authentication requests, see [Authenticating Requests (AWS Signature Version 4)](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
-    
-    When you select the authentication method, the applicable fields appear for the selected method. Enter the required details.
+    - **AWS Sigv4**: An AWS Signature Version 4 authenticating request is used to connect to the data source. AWS Sigv4 requires an access key and a secret key.
```
Comment (vagimeli) [inline _dashboards/discover/multi-data-sources.md:45]:
```suggestion
    - **AWS Sigv4**: An AWS Signature Version 4 authenticating request is used to connect to the data source. AWS SigV4 requires an access key and a secret key.
```

File: _dashboards/discover/multi-data-sources.md [hunk 2]
```diff
@@ -39,16 +39,21 @@ To create a new data source connection:
...
+      For SigV4 authentication, first specify the **Region**. Next, select the type of service in the **Service Name** list. The options are **Amazon OpenSearch Service** and **Amazon OpenSearch Serverless**. Last, enter the **Access Key** and **Secret Key** for authorization. For an example setup, see the following image.
 
-4. Select **Create data source connection** to save your settings. The connection is created. The active window returns to the **Data Sources** main page, and the new connection appears in the list of data sources.
+      <img src="{{site.url}}{{site.baseurl}}/images/dashboards/SigV4-setup.png" alt="SigV4 auth type setup" width="40%">
+      
+      For information about available AWS Regions for AWS accounts, see [Available Regions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions). For more about Sigv4 authentication requests, see [Authenticating Requests (AWS Signature Version 4)](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
```
Comment (vagimeli) [inline _dashboards/discover/multi-data-sources.md:51]:
```suggestion
      For information about available AWS Regions for AWS accounts, see [Available Regions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions). For more about SigV4 authentication requests, see [Authenticating Requests (AWS Signature Version 4)](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html).
```

File: _dashboards/discover/multi-data-sources.md [hunk 3]
```diff
@@ -58,9 +63,9 @@ To make changes to the data source connection, select a connection in the list o
...
 
-When **AWS Sigv4** is the selected authentication method, you can update the credentials by selecting **Update stored AWS credential**. In the pop-up window, enter a new  access key in the first field and a new secret key in the second field. Select **Update stored AWS credential** in the pop-up window. The new credentials are saved. Select **Test connection** in the upper-right corner of the screen to confirm that the connection is valid.
+When **AWS Sigv4** is the selected authentication method, you can update the credentials by selecting **Update stored AWS credential**. In the pop-up window, enter a new access key in the first field and a new secret key in the second field. Select **Update stored AWS credential** in the pop-up window. The new credentials are saved. Select **Test connection** in the upper-right corner of the screen to confirm that the connection is valid.
```
Comment (vagimeli) [inline _dashboards/discover/multi-data-sources.md:68]:
```suggestion
When **AWS SigV4** is the selected authentication method, you can update the credentials by selecting **Update stored AWS credential**. In the pop-up window, enter a new access key in the first field and a new secret key in the second field. Select **Update stored AWS credential** in the pop-up window. The new credentials are saved. Select **Test connection** in the upper-right corner of the screen to confirm that the connection is valid.
```

**Comment by vagimeli** [comment]
> Will replace image once UI text is fixed.

Here's suggested UI text if needed.

Enter the authentication information required to access the endpoint. If no authentication is required, select **No authentication**.

> Will replace image once UI text is fixed.

**Comment by cwillum** [comment]
We are aware that the text in the screen shot is missing a period at the end of the second sentence. This will be fixed in the future, and I'll swap out the image then.

**Review by natebower — COMMENTED** [review]
@cwillum Please see my comments and changes and let me know if you have any questions. Thanks!

**Pull request merged**